drain-cleaner/modules/drain-cleaner/install.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 set -e
 
-SCRIPT_DIR=$(dirname $0)
```

Comment:
Can this safety be removed? Isn't this used below?

Reply:
Fixed